### PR TITLE
Patient picture

### PR DESCRIPTION
**Pull request by MatthewVita**
## Purpose

I wanted to get the baseline work done for https://github.com/openemr/openemr/issues/593 so that Jeffrey can better understand how to tackle the problem. Once this is checked in, he can polish it (for example, the -5px margin is not very elegant and he can work on refreshing the image when a user uploads a new one!).

It's ready to go and once it's checked in I'll post something on Twitter with a screenshot saying that **we are working on making the EMR experience more personable.**

## Screenshots

Here's some screenshots from my test.

Latest chrome showing a real image with the compact mode on:

This is the latest IE showing the fallback avatar (I put this in the images folder and I made it myself so we don't have to deal with image copyrights):

Here's what happens when a patient isn't in context:

Nothing is broken when tabs mode is off:

### CODE REVIEW

**Comment by MatthewVita** [comment]
Please note that the image is 55px tall and the width will be scaled proportionally.

**Comment by MatthewVita** [comment]
@TheToolbox Looks like the PHPCS task failed. What is the best way to run the sniffer?

**Comment by TheToolbox** [comment]
This is what I've been running, which is close to what CI is doing:
```bash
composer global require "squizlabs/php_codesniffer=3.0.*"
~/.config/composer/vendor/bin/phpcs -pn --extensions=php,inc --standard=ci/phpcs.xml --report=summary
```

File: interface/main/tabs/templates/patient_data_template.php [hunk 1]
```diff
@@ -19,6 +19,15 @@
             <span style="float:left;"><a data-bind="click: viewPtFinder" href="#" class="btn btn-default btn-sm">
                 <i class="fa fa-search" aria-hidden="true"></i>
             </a></span>
+            <div class="patientInfo" style="float: left">
+              <!-- ko if: patient -->
+                  <div data-bind="with: patient" class="patientPicture">
+                      <img data-bind="attr: {src: patient_picture()}"
+                           alt="<?php echo xla("Patient Photograph") ?>"
+                           onError="this.src = '../../../images/patient-picture-default.png'" />
```
Comment (bradymiller) [inline interface/main/tabs/templates/patient_data_template.php]: should store the empty image in public/images/

Comment (bradymiller) [inline interface/main/tabs/templates/patient_data_template.php]: and this means for the path do:
``````
onError="this.src = '<?php echo $GLOBALS['images_static_relative']; ?>/patient-picture-default.png'" />

**Comment by bradymiller** [comment]
@MatthewVita ,
Very nice! The only minor issue is storing the empty image in public/images/ via $GLOBALS['images_static_relative'] (goal is to use the location for image in the future). Then this is ready to come into the codebase.
And the tweet sounds like a good idea.

**Comment by bradymiller** [comment]
@TheToolbox @MatthewVita 
for the CI and phpcs, here's a good wiki page that could be used:
http://www.open-emr.org/wiki/index.php/Continuous_Integration

(I'm actually working on bringing several of the larger projects into most recent codebase(esp medex and weno), and need to basically duplicate the phpbcs run that Robert did to do this efficiently; turns out I'll likely need to run the same exact version Robert did, will keep folks updated on when get a successful strategy for this)

**Comment by MatthewVita** [comment]
@bradymiller Thank you very much for the code review. Will fix that up and merge.

@TheToolbox Cool. Will update the wiki with this!

**Pull request merged**